### PR TITLE
Add new endpoints in .39

### PR DESCRIPTION
**Pull request by dragonsahead**
Added the documentation for the new endpoints of release 39

### CODE REVIEW

**Review by flamber — APPROVED** [review]
Probably want to wait until #15780 has been merged (which should go into release branch too)

**Comment by flamber** [comment]
Don't think you got the changes from #15780

**Comment by paoliniluis** [comment]
You're right @flamber, I didn't run the api-docs again, just did a new PR

**Pull request merged**
